extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.Optimizely.js

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Utility functions for running Optimizely A/B tests with VisualEditor
+ */
+
+/**
+ * Gets Optimizely ID for Toolbar A/B test
+ * @returns {string}
+ */
+ve.init.wikia.getToolbarABTestId = function () {
+	return window.wgDevelEnvironment ? '4721410313' : '4701112678';
+};
+
+/**
+ * @returns {number} The A/B test variant number for the Toolbar A/B test
+ */
+ve.init.wikia.getToolbarABTestVariantNumber = function () {
+	var optimizely = window.optimizely,
+		optimizelyId = ve.init.wikia.getToolbarABTestId();
+
+	if ( optimizely && optimizely.variationMap && optimizely.variationMap.hasOwnProperty( optimizelyId ) ) {
```

Comment:
Not super important but I would consider adding method ```getVariation``` or so directly in OptimizelyBlocking.js - so we likely would avoid code duplication in the future.